Components/badge/badge.css

```diff
@@ -0,0 +1,77 @@
+@import url(../avatar/avatar.css);
```

Comment:
You don't need to import css file separately, you can import it to component.css directly.

Reply:
I think like if I need somewhere only badge component means I can easily import badge.css, so it would be lightweight.
Is that ok?

Reply:
Ok, will change that